docs/reinit.md

```diff
@@ -0,0 +1,132 @@
+
+```
+$ kubectl get cm cluster1-instance1-24b8-config
+NAME                             DATA   AGE
+cluster1-instance1-24b8-config   1      95m
+```
+
+## Reinitializing by `patronictl reinit`
+
+You can reinitialize a replica using `patronictl reinit`. Note that configuration in ConfigMap might not have applied to running Patroni instance. It's recommended to run `patronictl reload` before running `patronictl reinit`.
+
+For example:
+
+```
+$ kubectl exec -it cluster1-instance1-24b8-0 -- cat /etc/patroni/~postgres-operator_instance.yaml
+# verify the configuration
+
+$ kubectl exec -it cluster1-instance1-24b8-0 -- patronictl reload cluster1-ha cluster1-instance1-24b8-0
++ Cluster: cluster1-ha (7487948770079264836) -------------------------+---------+-----------+----+-----------+
+| Member                    | Host                                    | Role    | State     | TL | Lag in MB |
++---------------------------+-----------------------------------------+---------+-----------+----+-----------+
+| cluster1-instance1-24b8-0 | cluster1-instance1-24b8-0.cluster1-pods | Replica | streaming |  1 |         0 |
+| cluster1-instance1-84xm-0 | cluster1-instance1-84xm-0.cluster1-pods | Leader  | running   |  1 |           |
+| cluster1-instance1-nv28-0 | cluster1-instance1-nv28-0.cluster1-pods | Replica | streaming |  1 |         0 |
++---------------------------+-----------------------------------------+---------+-----------+----+-----------+
+Are you sure you want to reload members cluster1-instance1-24b8-0? [y/N]: y
+Reload request received for member cluster1-instance1-24b8-0 and will be processed within 10 seconds
+
+$ kubectl exec -it cluster1-instance1-24b8-0 -- patronictl reinit cluster1-ha cluster1-instance1-24b8-0
++ Cluster: cluster1-ha (7487948770079264836) -------------------------+---------+-----------+----+-----------+
+| Member                    | Host                                    | Role    | State     | TL | Lag in MB |
++---------------------------+-----------------------------------------+---------+-----------+----+-----------+
+| cluster1-instance1-24b8-0 | cluster1-instance1-24b8-0.cluster1-pods | Replica | streaming |  1 |         0 |
+| cluster1-instance1-84xm-0 | cluster1-instance1-84xm-0.cluster1-pods | Leader  | running   |  1 |           |
+| cluster1-instance1-nv28-0 | cluster1-instance1-nv28-0.cluster1-pods | Replica | streaming |  1 |         0 |
++---------------------------+-----------------------------------------+---------+-----------+----+-----------+
+Are you sure you want to reinitialize members cluster1-instance1-24b8-0? [y/N]: y
+Success: reinitialize for member cluster1-instance1-24b8-0
+```
+
+## Configuring create_replica_methods
+
+If you want to change `create_replica_methods` list for any reason you can use `spec.patroni.create_replica_methods` field:
```

Comment:
Do we have pgBackRest set as the first one by default? Thinking of adding this info before "If you need to change the methods list"